indico/modules/events/registration/client/js/form/fields/InputCheckbox.jsx

```diff
@@ -0,0 +1,36 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2021 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import '../../../styles/regform.module.scss';
+
+export default function InputCheckbox({htmlName, disabled, title, isRequired}) {
+  return (
+    <div className="checkbox-with-text">
+      <input type="checkbox" name={htmlName} disabled={disabled} />
+      <div>
+        <span styleName="label">
```

Comment:
Let's make this an actual `<label>` so clicking it will toggle the checkbox. For the ID to link them you can use something like `regform-checkbox-${id}` - I think we pass the ID to this component,